Virtual Librarian/VirtualLibrarian.API/Controllers/AdminsController.cs

```diff
@@ -8,18 +8,17 @@ namespace VirtualLibrarian.API.Controllers
 {
     public class AdminsController : ApiController
     {
-
-        private static Library instance = null;
-        public AdminsController()
+        private readonly ILibrary _library;
```

Comment:
You are the first team whose code I am reviewing that have proper dependency injection. Good job :)

Reply:
One minor thing. `ILibrary` sounds a little bit too generic and a bit ambiguous. Wouldn't it clarify a bit if you named it `ILibraryManager`? Just a suggestion

Reply:
Thank you for your time :)